src/main/java/com/project/eventapp/controller/UserController.java

```diff
@@ -0,0 +1,40 @@
+package com.project.eventapp.controller;
+
+
+import com.project.eventapp.model.User;
+import com.project.eventapp.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.persistence.GeneratedValue;
+import java.util.List;
+
+@Controller
+@RequestMapping("/user")
+public class UserController {
+    private UserRepository userRepository;
+
+    public UserController(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @GetMapping("/users")
```

Comment:
@RequestMapping("/users") to tutaj nie potrzebujemy dodatkowych parametrów do mapowania. Czyli samo @GetMapping